src/main/java/no/sikt/generator/OpenApiCombiner.java

```diff
@@ -118,8 +117,8 @@ private void removeTags(OpenAPI api) {
 
     private void mergeSecurity(OpenAPI api) {
         if (nonNull(api.getSecurity())) {
-            for (SecurityRequirement securityRequirement : api.getSecurity()) {
-                logger.info("adding security req {}", securityRequirement.toString());
+            for (var securityRequirement : api.getSecurity()) {
```

Comment:
This was a cheap trick to avoid the prescription to use the interface, which here would have been quite harmful. It is not just us that makes things difficult on ourselves.